### ISSUE TITLE
I Cant not get Order done in any way!!

### ISSUE THREAD

**Issue by andrex16** [inspireui/support#3251]
Step 1 (require): detail issues and screenshots
I can ot get my order done in ay way, 
My payments methods loads from my wocommerce page perfectly, but when i choose any of both takes me to page not found error in my browser.

My question is, if its not configured by default, how can i get at least one order sent to my wocommerce site?, and how can I configure my second payment method that is Mercado Pago, (which are already configurated in my woocommerce site)

![Screenshot_20200517_203609_com inspireui fluxstore](https://user-images.githubusercontent.com/58792425/82167040-0d9f4900-9880-11ea-96a9-f1e164adea75.jpg)

Step 2 (require): verify license and sharing config files via this google form - http://private.inspireui.com
![Capturaas](https://user-images.githubusercontent.com/58792425/82167579-e3e72180-9881-11ea-82d6-17ec393d4b0a.JPG)
![Capturaas](https://user-images.githubusercontent.com/58792425/82167624-04af7700-9882-11ea-9b29-2486f2a43548.JPG)

 (please don't share the screenshot purchase here)

*Important Note:* 
- Please help to fill the required information from the above 2 steps and don't share the public screenshot purchase here, we may close the ticket which is not included the proof of purchase. Kindly help us to create only 1-Ticket & includes all the issues, that would help us focus to resolve it better. 
- If your all ticket was resolved & closed, but would like to create a new ticket, then you don't need to submit the form again, just do simple step by linking to the previous verified ID, example: #Ticket-ID
Thank you so much for your time 😊

**Comment by minhtruong315** [comment]
Hi @andrex16 I don't see not found page. https://www.dropbox.com/s/vlij4jxqdxznt7f/checkout.mov?dl=0
Can you record screen?

**Comment by andrex16** [comment]
here you have my screen recorded:
https://drive.google.com/file/d/1SzV8tAOkZsi0RI54I0f53LJcJorb5cRf/view?usp=sharing

and this is the redirect url that is not found 
https://muranosteel.com/checkout?code=a0ff8653f87b663e05f5f02b50d6c2fc&mobile=true


I thinnk that's so weird because on my side I get page not found error. Can you tell what can I do to solve this problem?

And after solving this issue if i want to make the whole payment only in the app whit out redirecting to my website becasue that looks more profesional.
 
Do I have to develope the payment method? (I am using Mercado Pago).

You should give it a try because that payent method its so common here in latin America.

And also to simply send the order directly to woocommerce whit out redirecting what do i have to do?, because this is also not working.

thanks in advance for yout help!

**Comment by minhtruong315** [comment]
@andrex16 you can disable these in lib/common/config/payments.dart
```
/// Enable Payment option
  "EnableOnePageCheckout": false,
  "NativeOnePageCheckout": false,
```

**Comment by andrex16** [comment]
Already tried, and also not working

**Comment by andrex16** [comment]
https://drive.google.com/file/d/1Z3mx35KwaEZ-WsUxyJX_ylDBo_lYQQXa/view?usp=sharing
https://drive.google.com/file/d/17GdWTrViOpHRFmd_Bm7MlwhtT8SSfFHJ/view?usp=sharing

**Comment by minhcasi** [comment]
@andrex16 - it seems the CloudFlare firewall setting to access from your website server, please help to unblock the access from the CloudFlare admin

**Comment by minhcasi** [comment]
I'm sorry to close this ticket as there is no response or active conversation, please help to create a new one and link to this ticket if the problem is not solved - thank you for your time!
